src/components/sustainability/HeroSection.tsx

```diff
@@ -21,7 +21,7 @@ const FeatureSection = ({ textContent }: HeroSectionsProps) => {
         {/* Main title */}
         <div className="flex flex-col items-center justify-center space-y-10 px-6 text-center">
           <Header>
-            <span className="text-green">{textContent.title.greenText}</span>
+            <span className="text-green-1">{textContent.title.greenText}</span>
```

Comment:
Does the text-green-1 class exist?

Reply:
Yes, i remade the tailwind for greens color range